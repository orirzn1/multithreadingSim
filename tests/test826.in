L9
S8
O8
N6

T0
I@0x00000000
ADD $0, $5, $2
SUBI $4, $7, 0x1
STORE $2, $0, 0x3
SUB $5, $1, $1
SUB $3, $7, $4
ADD $6, $4, $3
SUBI $6, $0, 0x1
SUB $4, $7, $1
ADDI $3, $3, 0x4
HALT $0

T1
I@0x00000000
SUB $2, $0, $1
ADD $7, $7, $4
LOAD $7, $1, $4
ADD $4, $7, $6
ADDI $3, $3, 0x4
ADD $4, $6, $5
ADDI $6, $7, 0x2
HALT $0

T2
I@0x00000000
SUBI $2, $7, 0x2
ADD $4, $4, $2
STORE $1, $6, $2
SUB $4, $6, $7
SUB $2, $6, $3
LOAD $3, $7, $4
SUBI $0, $7, 0x2
STORE $2, $7, 0x1
STORE $6, $3, $1
SUB $4, $6, $5
HALT $0

T3
I@0x00000000
ADDI $1, $6, 0x2
SUBI $7, $5, 0x1
SUBI $0, $6, 0x4
LOAD $6, $2, 0x4
ADD $7, $2, $7
SUBI $5, $7, 0x2
SUB $0, $7, $6
SUBI $0, $6, 0x4
HALT $0

T4
I@0x00000000
LOAD $4, $7, 0x1
ADDI $2, $1, 0x2
ADDI $4, $2, 0x3
LOAD $0, $5, $5
ADDI $2, $5, 0x3
SUBI $4, $7, 0x2
ADDI $3, $0, 0x4
SUBI $0, $4, 0x3
ADDI $4, $3, 0x1
HALT $0

T5
I@0x00000000
STORE $0, $7, 0x1
STORE $6, $5, $4
LOAD $5, $7, 0x1
SUB $5, $3, $3
ADD $2, $0, $2
HALT $0

D@0x0
0x3
0xA
0x7
0x7
0x8
0x8
0x3
0x9
0x1
0x3
