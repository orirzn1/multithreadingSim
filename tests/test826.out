
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x5	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFF	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x4	R4 = 0x0	R5 = 0x0	R6 = 0x2	R7 = 0x0
Register file thread id 2:
	R0 = 0xFFFFFFFE	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x6	R1 = 0x2	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFD	R6 = 0xA	R7 = 0xFFFFFFFF
Register file thread id 4:
	R0 = 0xFFFFFFFB	R1 = 0x0	R2 = 0x3	R3 = 0x4	R4 = 0x5	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Blocked MT CPI for this program 3.518519

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x5	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFF	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x4	R4 = 0x0	R5 = 0x0	R6 = 0x2	R7 = 0x0
Register file thread id 2:
	R0 = 0xFFFFFFFE	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x6	R1 = 0x2	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFD	R6 = 0xA	R7 = 0xFFFFFFFF
Register file thread id 4:
	R0 = 0xFFFFFFFB	R1 = 0x0	R2 = 0x3	R3 = 0x4	R4 = 0x5	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Finegrained Multithreading CPI for this program 1.203704

